### PR TITLE
Add SectionsAxis to combine symbol axis and number axes into one

### PR DESCRIPTION
**Pull request by Yuri-Blankenstein-TNO**
value-axis

### CODE REVIEW

File: src/main/java/org/jfree/chart/axis/SectionsAxis.java [hunk 1]
```diff
@@ -0,0 +1,788 @@
...
+import org.jfree.chart.event.AxisChangeEvent;
+import org.jfree.chart.event.AxisChangeListener;
+import org.jfree.chart.plot.PlotRenderingInfo;
+import org.jfree.chart.plot.ValueAxisPlot;
+import org.jfree.chart.text.TextUtils;
+import org.jfree.chart.ui.RectangleEdge;
+import org.jfree.chart.ui.TextAnchor;
+import org.jfree.chart.util.Args;
+import org.jfree.data.Range;
+
```
Comment (martijnhendrikstno) [inline src/main/java/org/jfree/chart/axis/SectionsAxis.java:70]: A high-level description of what this type does would be nice.

**Review by martijnhendrikstno — APPROVED** [review]

File: src/main/java/org/jfree/chart/axis/SectionsAxis.java (hunk 1 above)
Comment (Yuri-Blankenstein-TNO) [reply src/main/java/org/jfree/chart/axis/SectionsAxis.java:70]: I see that JavaDoc is missing for other methods as well. I'll add it.

**Review by martijnhendrikstno — APPROVED** [review]
Looks fine!

**Comment by Yuri-Blankenstein-TNO** [comment]
We decided to Move the SectionAxis to ESI (TNO) JFreeChart Extensions project. The SectionAxis is not mature enough to be added to JFreeChart. It would generate too many support requests. Therefor this merge request only contains the adaptations to JFreeChart that are required to implement such a SectionAxis.

**Pull request merged**